src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -757,12 +762,25 @@ static function (ChildDefinition $definition, AsPeriodicTask|AsCronTask $attribu
                 }
             );
         }
-        $container->registerAttributeForAutoconfiguration(JsonStreamable::class, static function (ChildDefinition $definition, JsonStreamable $attribute): void {
-            $definition->addTag('json_streamer.streamable', [
+
+        $container->registerForAutoconfiguration(CompilerPassInterface::class)
+            ->addExcludeTag('container.excluded.compiler_pass');
+        $container->registerForAutoconfiguration(KernelInterface::class)
+            ->addExcludeTag('container.excluded.kernel');
+        $container->registerForAutoconfiguration(TestCase::class)
+            ->addExcludeTag('container.excluded.test_case');
+        $container->registerAttributeForAutoconfiguration(AsMessage::class, static function (ChildDefinition $definition) {
+            $definition->addExcludeTag('container.excluded.messenger.message');
+        });
+        $container->registerAttributeForAutoconfiguration(Entity::class, static function (ChildDefinition $definition) {
+            $definition->addExcludeTag('container.excluded.doctrine.entity');
```

Comment:
For doctrine-bundle instead?

Reply:
If we do that (which is of course possible), it would make recipes much harder as we would need to deal with 2 versions. That's why having it here is a good compromise (by the way, we are doing the same here for PHPUnit for the same reasons).